### ISSUE TITLE
Export as image

### ISSUE THREAD

**Issue by Catnap1** [Steema/TeeChart-NET-Pro-Samples#2]
Hello, I am a beginner with TeeChart, and I am currently facing some difficulties in using TeeChart-Net. My English is not very good, so please forgive me. Here are the details:

I want to try exporting vector graphics in my code, as follows:

//string imagePath = ConstantUtil.AppFilePath + "temp\\temp_report_chart_" + DateTime.Now.Ticks + cnt.ToString() + ".wmf";
//MetafileFormat metafile = chart.Export.Image.Metafile;

//metafile.Height = 370;
//metafile.Width = 758;
//metafile.Save(imagePath);

Hello, I am a beginner with TeeChart, and I am currently facing some difficulties in using TeeChart-Net. My English is not very good, so please forgive me. Here are the details:

I want to try exporting vector graphics in my code, as follows:

csharp
Copy code
//string imagePath = ConstantUtil.AppFilePath + "temp\\temp_report_chart_" + DateTime.Now.Ticks + cnt.ToString() + ".wmf";
//MetafileFormat metafile = chart.Export.Image.Metafile;

//metafile.Height = 370;
//metafile.Width = 758;
//metafile.Save(imagePath);
However, I have encountered an issue. When exporting in BMP format, the line thickness is normal. But when exporting in vector graphic format (WMF), the lines become thicker, and even when I modify the line thickness property, it doesn't change.

So, how should I handle this to prevent the lines from becoming too thick in WMF format?

Thank you for your assistance.

**Comment by Catnap1** [comment]
To add, it is not exported in the [editor] editing interface, but written and exported in the code.

**Comment by mogmem** [comment]
Hi, I've just run a test on that using your codelines. I'm seeing a faithful reproduction of line width in the saved wmf file. What version (build) of TeeChart NET are you using?

**Comment by Catnap1** [comment]
@mogmem  Hi, I am using version of TeeChart NET is "4.2021.8.23", Is this possible due to the version?

**Comment by mogmem** [comment]
No, that version should be fine. I'm unable to reproduce the issue. If you can send a small sample project and a screenshot of what you're seeing to info@steema.com we'll take a closer look. Thanks!
